test/functional/specs/Data Collector/C8118.js

```diff
@@ -67,11 +67,11 @@ test("Test C8118: Verify link click sends a request to the collect endpoint when
   // the fetch API in this case and there is no gurantee that the browser will
   // finish sending the request before navigating the user.
 
-  await t.expect(networkLogger.edgeCollectEndpointLogs.requests.length).eql(0);
-  await assertRequest(networkLogger.edgeInteractEndpointLogs.requests[0]);
-  if (isSendBeaconSupported()) {
-    await t.expect(sendBeaconMock.getCallCount()).eql(0);
-  }
+  // await t.expect(networkLogger.edgeCollectEndpointLogs.requests.length).eql(0);
+  // await assertRequest(networkLogger.edgeInteractEndpointLogs.requests[0]);
+  // if (isSendBeaconSupported()) {
+  //   await t.expect(sendBeaconMock.getCallCount()).eql(0);
+  // }
```

Comment:
@shammowla did you intend to leave this commented out?

Reply:
Yeah, we commented this part out together.

  // We cannot assert the interact endpoint has been called because Alloy uses
  // the fetch API in this case and there is no guarantee that the browser will
  // finish sending the request before navigating the user.

Reply:
Let's just comment out this line only:

```
await assertRequest(networkLogger.edgeInteractEndpointLogs.requests[0]);
```